src/main/resources/db/changelog/changes/01-create-books-table.yml

```diff
@@ -39,7 +39,7 @@ databaseChangeLog:
                   type: varchar(500)
               - column:
                   name: cover_image
-                  type: varchar(500)
+                  type: varchar(255)
```

Comment:
not sure why would we need to reduce it?

Reply:
It's was for liquibase fix in my pc

Reply:
let's not change it to 255. 
If it would be a real project, liquibase scripts that were already pushed to master(and thus deployed to some live server) would already be applied on the server and on your colleagues PC's, and if you were to merge this change here - you would have to rollback this script on the staged environments and your colleagues would have to do that on their PC's as well, which is a whole lot of trouble.
What we should do in such situation: rollback this script/clear the DB on your local machine, and run application on the main branch, to ensure that your machine can work properly with the scripts that you have in master